ci/code_checks.sh

```diff
@@ -576,7 +576,9 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
     $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=EX02 --ignore_functions \
         pandas.DataFrame.plot.line \
         pandas.Series.plot.line \
-        pandas.Timestamp.fromtimestamp \
+        pandas.Series.sparse.density \
+        pandas.Series.sparse.npoints \
+        pandas.Series.sparse.sp_values \
```

Comment:
is this intentional? looks like you're adding `pandas.Series.sparse.sp_values` to the ignore list - perhaps something didn't go quite right when merging?

Reply:
It wasn't intentional. I am not very familiar with resolving merge conflicts. 

While I was working on the PR the functions `pandas.Series.sparse.density`, `pandas.Series.sparse.npoints`, `pandas.Series.sparse.sp_values` were in the ignore list. I deleted these functions from the ignore list in my next PR, which has been merged just a couple of hours ago. To resolve the conflict I have to delete these functions now.

I can do it locally and push a new commit. What do you think?

Reply:
sounds good, thanks!